### PR TITLE
jwk #8: jwk consensus wired into node

### PR DESCRIPTION
**Pull request by zjma**

### CODE REVIEW

**Review by grao1991 — APPROVED** [review]

File: aptos-node/src/lib.rs [hunk 1]
```diff
@@ -689,24 +689,32 @@ pub fn setup_environment_and_start_node(
...
-            network_client,
-            network_service_events,
-            vtxn_pool.clone(),
-            reconfig_events,
-            onchain_jwk_updated_events,
-        );
-        Some(jwk_consensus_runtime)
-    } else {
-        None
+    let maybe_jwk_consensus_key =
```
Comment (danielxiangzl) [inline aptos-node/src/lib.rs:692]: Do you need to load twice? Seems same as dkg

File: aptos-node/src/lib.rs [hunk 2]
```diff
@@ -689,24 +689,32 @@ pub fn setup_environment_and_start_node(
...
+        (Some(interfaces), Ok(consensus_key)) => {
+            let ApplicationNetworkInterfaces {
+                network_client,
+                network_service_events,
+            } = interfaces;
+            let (reconfig_events, onchain_jwk_updated_events) = jwk_consensus_subscriptions.expect(
+                "JWK consensus needs to listen to NewEpochEvents and OnChainJWKMapUpdated events.",
+            );
+            let my_addr = node_config.validator_network.as_ref().unwrap().peer_id();
+            let jwk_consensus_runtime = start_jwk_consensus_runtime(
```
Comment (danielxiangzl) [inline aptos-node/src/lib.rs:706]: We don't check feature flag here?

File: aptos-node/src/lib.rs (hunk 1 above)
Comment (zjma) [reply aptos-node/src/lib.rs:692]: i don't want to, but the key is not clonable...

File: aptos-node/src/lib.rs (hunk 2 above)
Comment (zjma) [reply aptos-node/src/lib.rs:706]: it seems in the existing pattern,  it's epoch manager's job.

**Review by danielxiangzl — APPROVED** [review]

**Pull request merged**